src/components/Loading/Loading.js

```diff
@@ -34,16 +34,29 @@ export default class Loading extends React.Component {
      * Specify whether you would like the small variant of <Loading>
      */
     small: PropTypes.bool,
+
+    /**
+     * Specify a description that would be used to describe the loading state
```

Comment:
Maybe some more context here around what the end result of this prop actually does (updating the title of the svg)